examples/raylib/README.md

```diff
@@ -0,0 +1,15 @@
+A minimal example of using [zig-android-sdk](https://github.com/silbinarywolf/zig-android-sdk) to build raylib for android.
```

Comment:
Can you please update this readme to conform a bit more to the structure of the SDL2 example?